presto-parser/src/test/java/io/prestosql/sql/parser/TestSqlParser.java

```diff
@@ -1844,7 +1845,7 @@ public void testShowStats()
         final String[] tableNames = {"t", "s.t", "c.s.t"};
 
         for (String fullName : tableNames) {
-            QualifiedName qualifiedName = QualifiedName.of(Arrays.asList(fullName.split("\\.")));
+            QualifiedName qualifiedName = QualifiedName.of(Arrays.asList(fullName.split("\\.")).stream().map(Identifier::new).collect(Collectors.toList()));
```

Comment:
This is a little hard to read. I'd pull it out into a `makeQualifiedName`. Also, I'd structure it as:

```java
List<Identifier> parts = Arrays.stream(fullName.split("\\."))
        .map(Identifier::new)
        .collect(Collectors.toList());

return QualifiedName.of(parts);
```